x/emissions/Makefile

```diff
@@ -32,7 +32,7 @@ proto-gen:
 	@$(protoImage) sh ./scripts/protocgen.sh
 	@go mod tidy
 
-proto-format:
```

Comment:
should be on x/mint too?

Reply:
#584

Reply:
im just saying proto-fmt not in that one. minor